<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectLanguageWindowClinical</class>
 <widget class="QDialog" name="SelectLanguageWindowClinical">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="languageAccept">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>170</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QLabel" name="languageTitle">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>13</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#BDBDBD</string>
   </property>
   <property name="text">
    <string>Select Language</string>
   </property>
  </widget>
  <widget class="QPushButton" name="languageCancel">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>170</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>349</width>
     <height>213</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 49, 49);
border: 1px;
border-radius: 5px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>37</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="english">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#BDBDBD</string>
       </property>
       <property name="text">
        <string>English</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="chinese">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#BDBDBD</string>
       </property>
       <property name="text">
        <string>Chinese</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="japanese">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#BDBDBD</string>
       </property>
       <property name="text">
        <string>Japanese</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>languageCancel</zorder>
  <zorder>languageAccept</zorder>
  <zorder>languageTitle</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
